kong/api/routes/apis.lua

```diff
@@ -46,8 +46,10 @@ return {
     end,
 
     POST = function(self, dao_factory)
-      crud.post(self.params, dao_factory.plugins, function(data)
-        reports.send("api", utils.deep_copy(data))
+      crud.post(self.params, dao_factory.plugins, nil, function(data)
```

Comment:
+1 to removing the config. For the callback fix though, isn't it better to fix it by reverting this particular change:

https://github.com/Mashape/kong/commit/cc4ffc30397b0a1291c456f39840d88ba00f7b98#diff-241da6cd7103434c320de9b1240e7452L196

Which adds a new callback argument, but it is not being used.

If we go this way, the only place where `post()` is receiving 4 arguments is (I believe) here: https://github.com/Mashape/kong/commit/cc4ffc30397b0a1291c456f39840d88ba00f7b98#diff-9c0c4cb557f6a227f45c3eb87b3070e9R30